travis/test_server.py

```diff
@@ -133,6 +145,17 @@ def get_addons_path(travis_home, travis_build_dir, server_path):
     return addons_path
 
 
+def get_default_travis_addons_path():
+    '''
+
+    '''
+    return get_addons_path(
+        os.environ['HOME'],
+        os.environ['TRAVIS_BUILD_DIR'],
+        get_default_travis_server_path(),
+    )
+
+
```

Comment:
I don't see the point to have these two helper functions added here.
Why don't you add them to `getaddons.py`?